tests/site.yml

```diff
@@ -0,0 +1,14 @@
+- hosts: firewalld-servers
```

Comment:
And this?

Reply:
No, I added that when I was running Vagrant.  Should we include this in the event that someone wants to test with Vagrant, or is that redundant with molecule?